libparsec/crates/platform_storage/src/native/workspace.rs

```diff
@@ -641,6 +650,30 @@ async fn db_get_manifest(
     }
 }
 
+async fn db_list_manifests(
+    executor: impl sqlx::Executor<'_, Database = sqlx::Sqlite>,
+    offset: u32,
+    limit: u32,
+) -> anyhow::Result<Vec<RawEncryptedManifest>> {
+    let rows = sqlx::query("SELECT blob FROM vlobs LIMIT ?1 OFFSET ?2")
+        .bind(limit)
+        .bind(offset)
+        .fetch_many(executor);
+
+    let res: anyhow::Result<Vec<RawEncryptedManifest>> = rows
+        .filter_map(|row_res| async {
+            match row_res {
+                Ok(sqlx::Either::Left(_res)) => None,
```

Comment:
This `sqlx::Either` is pretty unclear, I had to do some hard digging into sqlx & sqlx-sqlite to properly understand it...

So I suggest adding a comment:
```rust
// `fetch_many` returns a single "left" element for the query result (containing, among others,
// the number of changes), then (what we are actually interested into here)  one "right"
// elements for each row in the result.
```

Reply:
done